### ISSUE TITLE
Pre-submission inquiry: multiple repositories for a single paper

### ISSUE THREAD

**Issue by JacksonMaxfield** [openjournals/joss#990]
Hello all!

We plan to submit a paper for a suite of tools:

- a backend library that stores database schema, processing functions, infrastructure setup, etc.
- a frontend library that stores the code for a web app to actually view and interact with the produced data
- a cookiecutter template to combine the two and get a whole new deployment of the software

I saw #565 and was wondering if that is still the go to method? For our case I think there is also a fair argument to be made for putting the paper in the cookiecutter repo itself. As it already ties the other libraries together but doesn't submodule the code into the generated repository / deployment.

For reference, here is a link to the cookiecutter that has many more details: https://github.com/CouncilDataProject/cookiecutter-cdp-deployment

In short:
1. is the preferred strategy for multi-repo submissions still "create a repo specifically for the paper and submodule in the others"?
2. in this case where we have a cookiecutter template, should we place the paper there?

**Comment by arfon** [comment]
@JacksonMaxfield – it's OK for the paper to be in one of the repositories (Cookiecutter is fine), but realize that the paper (and JOSS website) will link to that as the 'software repository' associated with the submission (this can't be multiple links sorry).

Then, all you need to do at submission/review time is make it clear that there are actually multiple packages that need reviewing here. At the end of the review, we could ask you to make a single archive of all of the source code for all of the repositories for archiving purposes.

While we discussed submodules in #565, I don't think this is necessary.

**Comment by JacksonMaxfield** [comment]
Thanks for the info @arfon!
